### ISSUE TITLE
Error message when cropping protected images

### ISSUE THREAD

**Issue by devsikander** [danmichaelo/croptool#119]
`Upload failed! [api] Received error: cascadeprotected : This page has been protected from editing because it is transcluded in the following page, which is protected with the "cascading" option turned on: * [[:Commons:Auto-protected files/wikipedia/en]]`

"Upload failed" is incorrect because the new cropped image did upload to Commons, but editing the source image to add the {{Image extracted}} information failed because of image page protection.

https://commons.wikimedia.org/wiki/File:Donna_Strickland_-_2017_(cropped).jpg from https://commons.wikimedia.org/wiki/File:Donna_Strickland.jpg

**Comment by danmichaelo** [comment]
Right, should be "Edit failed" or something more explanatory. Will fix
